### PR TITLE
Submission/Problem UI Fixes

### PR DESCRIPTION
**Pull request by gblake31**
1. Fixed Bug where refreshing the verdict page makes the problem name say 'Custom Submission'
    - I'm no longer loading the page or calling get_verdict() until the problem object returns with valid contents.
2. Now there's only one file on the front-end that describes what properties a Problem has.
3. Mouse over the individual test case icons to see their verdict as a tooltip
4. Time Limit, Memory Limit, Source, and Author now part of the problem block

https://github.com/ofast-team/frontend/assets/51721851/564940f7-eabd-4fbb-b270-53ab48d5aa25

### CODE REVIEW

File: src/pages/VerdictPage.tsx [hunk 1]
```diff
@@ -223,6 +240,7 @@ export default function VerdictPage() {
     window.URL.revokeObjectURL(link.href)
   }
 
+  console.log(testCases)
```
Comment (sd0623) [inline src/pages/VerdictPage.tsx:243]: remove

File: src/pages/VerdictPage.tsx [hunk 2]
```diff
@@ -343,15 +361,27 @@ export default function VerdictPage() {
...
-                return <CorrectIcon />
-              }
-              if (status > 3) {
-                return <WrongAnswerIcon />
-              }
-              return <PendingIcon />
-            })}
+            {testCases?.map((status) => (
+              <Tooltip
+                title={verdictInfo[status].description}
```
Comment (sd0623) [inline src/pages/VerdictPage.tsx:366]: I really like this. But how about different test cases with different verdicts. This is in regards to how Judge0 is sending the verdict. Cause if there is a wrong answer test case and time limit exceeded test case, and the overall verdict being only one of that, then that would be mean the tooltip for the other test case is incorrect.

**Review by sd0623 — COMMENTED** [review]
Looks good. Just left couple comments.

File: src/pages/VerdictPage.tsx (hunk 2 above)
Comment (gblake31) [reply src/pages/VerdictPage.tsx:366]: I don't think that's true. This tooltip should show each unique test case's 'verdict', unless the code is wrong somewhere.

Comment (sd0623) [reply src/pages/VerdictPage.tsx:365]: alright I missed that, cool 👍

**Review by sd0623 — APPROVED** [review]

**Pull request merged**